app/services/jobs/return-logs/create-quarterly-return-cycles.service.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+/**
+ * Given a return cycle, generate each quarterly return period
+ * @module CreateQuarterlyReturnCycleService
+ */
+
+const { determineReturnsPeriods } = require('../../../lib/return-periods.lib.js')
+
+/**
+ * Given a return cycle, generate each quarterly return period
+ *
+ * @param {object} returnCycle - the return cycle to break into quarterly period
+ *
+ * @returns {Array<object>} the created quarterly return periods
+ */
+function go(returnCycle) {
+  if (!returnCycle) {
+    return []
+  }
```

Comment:
Is there ever a scenario we would not pass it a return cycle, and then happily deal with an empty result?

If we never pass it an empty cycle, we shouldn't have the guard clause to handle the scenario.